app/queries/servers/entry.ts

```diff
@@ -67,3 +80,38 @@ export async function prepareModels({operator, initialTeamId, removeTeams, remov
 
     return modelPromises;
 }
+
+export async function truncateCrtRelatedTables(serverUrl: string): Promise<{error: any}> {
+    const operator = DatabaseManager.serverDatabases?.[serverUrl].operator;
+
+    if (!operator) {
+        return {error: 'No App database found'};
+    }
+    const database = operator.database;
```

Comment:
there is a new function in the manager that will throw if not found so you can add it directly inside the try/catch block.

so you can do
```ts
const {database, operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
```